language-server/src/dotty/tools/languageserver/Worksheet.scala

```diff
@@ -31,59 +37,59 @@ object Worksheet {
                sendMessage: String => Unit,
                cancelChecker: CancelChecker)(
       implicit ctx: Context): Unit = {
+
+    val replMain = dotty.tools.repl.WorksheetMain.getClass.getName.init
+    val classpath = sys.props("java.class.path")
```

Comment:
Shouldn't the classpath used for the worksheet be the same classpath that the language server uses for the InteractiveDriver for the current directory ?

Reply:
This one is the JVM classpath that runs the REPL, the REPL receives `ctx.settings.classpath.value` as classpath.